docker/wallet/Dockerfile

```diff
@@ -28,7 +28,7 @@ WORKDIR /dist
 RUN cp /build/main .
 
 FROM scratch as dist
-
+COPY custom_account_setup.cdc ./
```

Comment:
Here we're copying the custom account creation script into the docker container so that it's available to use when creating new accounts.